tests/test_resolve.py

```diff
@@ -1004,20 +1004,20 @@ def test_no_features():
     ]
 
 
-@pytest.mark.skipif(datetime.now() < datetime(2018, 9, 1), reason="bogus test; talk with @mcg1969")
-def test_multiple_solution():
-    index2 = index.copy()
-    fn = 'pandas-0.11.0-np16py27_1.tar.bz2'
-    res1 = set([fn])
-    for k in range(1,15):
-        fn2 = Dist('%s_%d.tar.bz2'%(fn[:-8],k))
-        index2[fn2] = index[Dist(add_defaults_if_no_channel(fn))]
-        res1.add(fn2)
-    index2 = {Dist(key): value for key, value in iteritems(index2)}
-    r = Resolve(index2)
-    res = r.solve(['pandas', 'python 2.7*', 'numpy 1.6*'], returnall=True)
-    res = set([y for y in res if y.name.startswith('pandas')])
-    assert len(res) <= len(res1)
+#@pytest.mark.skipif(datetime.now() < datetime(2018, 9, 1), reason="bogus test; talk with @mcg1969")
```

Comment:
This was failing due to undefined imports @kalefranz 

`add_defaults_if_no_channel` and `Dist`

Reply:
Right, that's why the timebomb was there.  I don't want it just commented out.  Rebase against master and uncomment.

EDIT: Actually looks like you already did.  I extended all the timebombs to Sept 1 yesterday.